### ISSUE TITLE
The dependency "zod-validation-error" no longer supports Node.js v16

### ISSUE THREAD

**Issue by NicodeSS** [webpro/knip#321]
> Knip supports LTS versions of Node.js, and currently requires at least Node.js v16.17 or v18.6.

The dependency “zod-validation-error” no longer supports Node.js v16, preventing the installation of knip in a Node.js v16.17.1 environment.

Please consider update requirements or give a proper fix?

```
yarn add knip
yarn add v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
error zod-validation-error@2.0.0: The engine "node" is incompatible with this module. Expected version ">=18.0.0". Got "16.17.1"
error Found incompatible module.
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.38.3. See [Release 2.38.3](https://github.com/webpro/knip/releases/tag/2.38.3) for release notes._

**Comment by webpro** [comment]
Thanks for the report, @NicodeSS (they [really sneaked that in](https://github.com/causaly/zod-validation-error/releases/tag/v2.0.0))!
